libs/langchain/langchain/llms/anyscale.py

```diff
@@ -110,14 +112,31 @@ def _call(
             f"{self.anyscale_service_url}{self.anyscale_service_route}"
         )
         headers = {"Authorization": f"Bearer {self.anyscale_service_token}"}
-        body = {"prompt": prompt}
-        resp = requests.post(anyscale_service_endpoint, headers=headers, json=body)
-
-        if resp.status_code != 200:
-            raise ValueError(
-                f"Error returned by service, status code {resp.status_code}"
-            )
-        text = resp.text
+
+        if self.anyscale_service_url.startswith(ANYSCALE_ENDPOINT):
+            body = {
+                "model": "meta-llama/Llama-2-70b-chat-hf",
```

Comment:
if this is a chat model should be implemented as ChatModel. can create a ChatAnyscale(ChatModel) class. happy to help if needed

Reply:
Thanks for the suggest. I will close this PR for now and will work on it later